.gitignore

```diff
@@ -6,6 +6,7 @@ pom.xml.next
 release.properties
 dependency-reduced-pom.xml
 buildNumber.properties
+~$*
```

Comment:
👍 
Do you think we should add this one to the archetype as well, or is this just specific for this project since it is working with Excel/CSV?